app/controllers/deploys_controller.rb

```diff
@@ -139,6 +134,6 @@ def find_deploy
   end
 
   def active_deploy_scope
-    @project ? @project.deploys.active : Deploy.active
+    current_project ? current_project.deploys.active : Deploy.active
```

Comment:
Oh, sorry, my bad. I didn't see the `if params[:project_id]` conditional in `require_project`.  So I thought that calling `current_project` would either return a valid Project, or throw an error.

Nevermind, carry on.